### DISCUSSION TITLE
How should I convert a scanned PDF document to PDFA using gotenberg7?

### DISCUSSION THREAD

**Discussion by gloria256** [gotenberg/gotenberg#734] [Q&A]
Currently gotenbert7 allows the conversion of scanned documents to PDFA, but it happens that when opening the resulting PDFA it is distorted, as shown below:

![Prueba](https://github.com/gotenberg/gotenberg/assets/57493829/aff9690b-511a-4209-88ef-e14cbb243992)

I appreciate any ideas on how to solve it.

**Comment by robert-kaino** [comment]
Hi, @gloria256!

Did you find a solution to this one?  :) Struggling a bit with same situation and have tested with different fonts etc, but seems that some docs just not compatible with libreOffice draw :/
